asv/commands/run.py

```diff
@@ -499,10 +506,18 @@ def iter_rounds_commits():
 
                         if not skip_save:
                             result.save(conf.results_dir)
+                        if strict:
+                            failed = [
+                                failed_bench for failed_bench in
+                                result.errcode if
+                                result.errcode[failed_bench] != 0]
+                            failures += failed
```

Comment:
Maybe better to make `failures` to boolean and do `failures = failures or any(code != 0 for code in result.errcode.values())` -- storing the failed benchmarks in a list is not necessary, so best not do it.

Reply:
Fixed in: https://github.com/airspeed-velocity/asv/pull/865/commits/324ba8016bd9a4369a40d35a2b19a703852c9ab2